services/mock-service/src/fsp-integration/intersolve-visa/intersolve-visa.mock.service.ts

```diff
@@ -550,4 +550,62 @@ export class IntersolveVisaMockService {
       status: HttpStatus.OK,
     };
   }
+
+  public linkToken(
+    _childTokenCode: string,
+    parentTokenCode: string,
```

Comment:
Consider switching the parameters: first parent, then child. Feels more natural, and aligns with similar function in IntersolveVisaApiService. :)

Reply:
remove child token here